calico_node/filesystem/etc/calico/confd/templates/bird_ipam.cfg.template

```diff
@@ -2,20 +2,29 @@
 filter calico_pools {
   calico_aggr();
   custom_filters();
-{{range ls "/pool"}}{{$data := json (getv (printf "/pool/%s" .))}}
+{{range ls "/v1/ipam/v4/pool"}}{{$data := json (getv (printf "/v1/ipam/v4/pool/%s" .))}}
   if ( net ~ {{$data.cidr}} ) then {
     accept;
   }
 {{end}}
   reject;
 }
 
+{{$network_key := printf "/bgp/v1/host/%s/network_v4" (getenv "NODENAME")}}{{$network := getv $network_key}}
 filter calico_ipip {
-{{range ls "/pool"}}{{$data := json (getv (printf "/pool/%s" .))}}{{if $data.ipip}}
+{{range ls "/v1/ipam/v4/pool"}}{{$data := json (getv (printf "/v1/ipam/v4/pool/%s" .))}}
   if ( net ~ {{$data.cidr}} ) then {
+{{if eq $data.ipip_mode "cross-subnet"}}
+    if ( from ~ {{$network}} ) then
+      krt_tunnel = "";
+    else
+      krt_tunnel = "{{$data.ipip}}";
+    accept;
+  } {{else}}
     krt_tunnel = "{{$data.ipip}}";
```

Comment:
IIUC, this line will get executed even if `ipip` is not enabled, but when not enabled `$data.ipip` will be `""` and so traffic won't go over the tunnel?  Just trying to make sure I understand what I'm reviewing :)